tooling/templatize/pkg/pipeline/arm.go

```diff
@@ -273,32 +273,31 @@ func doWaitForDeployment(ctx context.Context, client *armresources.DeploymentsCl
 		if err != nil {
 			return nil, fmt.Errorf("failed to create deployment: %w", err)
 		}
-		logger.Info("Deployment started", "deployment", step.Name)
+		logger.V(1).Info("Deployment started", "deployment", step.Name)
 
 		return pollAndGetOutput(ctx, poller)
 	} else {
 		poller, err := client.BeginCreateOrUpdate(ctx, rgName, step.Name, deployment, nil)
 		if err != nil {
 			return nil, fmt.Errorf("failed to create deployment: %w", err)
 		}
-		logger.Info("Deployment started", "deployment", step.Name)
+		logger.V(1).Info("Deployment started", "deployment", step.Name)
 
 		return pollAndGetOutput(ctx, poller)
 	}
 }
 
 func (a *armClient) ensureResourceGroupExists(ctx context.Context, rgName string, rgNoPersist bool) error {
 	// Check if the resource group exists
-	// todo fill tags properly
-	tags := map[string]*string{}
-
-	if !rgNoPersist {
-		// if no-persist is set, don't set the persist tag, needs double negotiate, cause default should be true
-		tags["persist"] = to.Ptr("true")
-	}
-	_, err := a.resourceGroupClient.Get(ctx, rgName, nil)
```

Comment:
changed the tag handling so that once persist: true is set on a RG, it is not lifted by templatize anymore